app/views/resources/_thumbnail_display.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="row mb-3 no-gutters d-inline-block">
```

Comment:
Maybe this could be more generically renamed as `_heading_display.html.erb` or something like that, since it contains more than the thumbnail (title and description), and sometimes does not even contain the thumbnail.